lib/browser/storage.js

```diff
@@ -5,10 +5,18 @@ try {
   hasStorage = "localStorage" in window;
   // Attempt to access localStorage
   localStorage.length;
```

Comment:
I believe this access could be removed now, as it's superseded by our additions.